Swiftfin/Views/ConnectToServerView.swift

```diff
@@ -179,4 +162,25 @@ struct ConnectToServerView: View {
             L10n.serverAlreadyExistsPrompt(server.name).text
         }
     }
+
+    func handleConnection(_ event: ConnectToServerViewModel.Event) {
```

Comment:
Even thought not really necessary, I do like things `private`, and have `body` last. I think I'd like this moved above all `View` builders, so after `timer`.

```suggestion
    private func handleConnection(_ event: ConnectToServerViewModel.Event) {
```